index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
     if (err) return cb(err)
-    // Filter deletions & map to old expected format.
-    docs = filterObj(docs, function (key, value) {
-      return !value.deleted
-    })
-    docs = mapObj(docs, function (key, value) {
-      return value.value
+    docs = mapObj(docs, function (version, value) {
```

Comment:
I think I'd like to make this work (exposing setting/getting values on deletions) as a separate PR, to avoid adding even more complexity to this already large PR. We can hold off on the major semver until I get it in. Sounds okay?